docker-compose.override.yml

```diff
@@ -15,6 +15,7 @@ services:
     image: mysql:5.7
```

Comment:
Should this be MySQL 8, since that's what Matt is using?

Reply:
Yeah, I think so.